Products/Zuul/routers/triggers.py

```diff
@@ -129,8 +129,8 @@ def getNotification(self, uid):
     def updateNotification(self, **data):
         notificationUid = data['uid']
         response = self._getFacade().updateNotification(**data)
-        audit('UI.Notification.Edit', notificationUid,
-              data=data, maskFields='password')
+        audit('UI.Notification.Edit', object_=notificationUid,
+              data_=data, maskFields_='password')
```

Comment:
shouldn't maskFields_ be a list/tuple of strings ?

Reply:
Both are applicable 
https://github.com/zenoss/ZenPacks.zenoss.AuditLog/blob/b451aa5a1bf48cceec8ffdf1a70e377d9150eab8/ZenPacks/zenoss/AuditLog/manager.py#L175